ui/selectors/selectors.js

```diff
@@ -1874,20 +1821,14 @@ export const getMemoizedCurrentChainId = createDeepEqualSelector(
   (chainId) => chainId,
 );
 
-export const getMemoizedUnapprovedPersonalMessages = createDeepEqualSelector(
-  (state) => state.metamask.unapprovedPersonalMsgs,
-  (unapprovedPersonalMsgs) => unapprovedPersonalMsgs,
-);
-
-export const getMemoizedUnapprovedTypedMessages = createDeepEqualSelector(
-  (state) => state.metamask.unapprovedTypedMessages,
-  (unapprovedTypedMessages) => unapprovedTypedMessages,
-);
-
 export function getSnaps(state) {
   return state.metamask.snaps;
 }
 
+export function testFunctionToSeeIfItsCaught(state) {
+  return state.metamask.snaps;
+}
```

Comment:
Is this needed, it looks same as `getSnaps` above ?

Reply:
This was committed by mistake, great catch Jyoti. I'll remove this now.